toptal/picasso
# fix(TopBarMenu): [TEA-2354] Improve responsiveness

[TEA-2354]

### Description

At Talent Portal, after migrating to V5 we faced a few issues in header for small screens and this PR fix them:
- fix sidebar menu icon position
- fix user dropdown avatar size

### How to test

- Check if sidebar menu icon is aligned with left margin of page
- Check if user dropdown has proper size for small screen

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| <img width="374" alt="Screenshot 2021-03-08 at 15 08 08" src="https://user-images.githubusercontent.com/7594216/110332145-416e7980-8020-11eb-93ef-2cd8a3629c39.png">| <img width="374" alt="Screenshot 2021-03-08 at 15 09 05" src="https://user-images.githubusercontent.com/7594216/110332180-4d5a3b80-8020-11eb-9061-e750f720f432.png"> |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[TEA-2354]: https://toptal-core.atlassian.net/browse/TEA-2354
